test/automation/src/code.ts

```diff
@@ -254,6 +254,10 @@ export class Code {
 		return this.driver.getLogs();
 	}
 
+	wait(millis: number): Promise<void> {
```

Comment:
Can we adopt this in places where we use `new Promise(resolve => setTimeout...` today? For example there is a use in this file already.

Reply:
Adopted